File: C/Palindrome.c
#include<stdio.h>
void main()
{
	int w,n,d,s=0;
	printf("Enter a number : ");
	scanf("%d",&n);
	w=n;
	while(w!=0)
	{
		d=w%10;
		s=s*10+d;
		w/=10;
	}
	if(s==n)
	printf("Palindrome number");
	else
	printf("Not a Palindrome number");
}

File: Java/Practice_programs/Palindrome.java
import java.io.*;

public class Palindrome
{
    public static void main(String args[]) throws IOException
    {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        System.out.print("Enter a string: ");
        String s = br.readLine();
        StringBuffer ns = new StringBuffer();
        ns.append(s);
        ns.reverse();
        if (s.equals(ns.toString()))
            System.out.println("Palindrome string");
        else
            System.out.println("non-Palindrome string");
    }
}
